package.json

```diff
@@ -19,14 +19,15 @@
     "clean": "rm -rf dist"
   },
   "dependencies": {
-    "soap": "^0.30.0",
-    "tslint": "^5.16.0",
-    "typescript": "^3.4.5",
-    "xml2js": "^0.4.19"
+    "soap": "^0.45.0",
+    "tslint": "^6.1.3",
+    "typescript": "^4.8.2",
+    "xml2js": "^0.4.23"
   },
   "devDependencies": {
-    "@types/bluebird": "^3.5.26",
-    "@types/node": "^13.1.4",
-    "@types/xml2js": "^0.4.4"
+    "@types/bluebird": "^3.5.36",
+    "@types/node": "^18.7.14",
+    "@types/sax": "^1.2.4",
```

Comment:
is probably an accident that slipped in here. Or did you feel a benefit from it?

Reply:
Oh yeah my bad, should've posted the reason. This seems to be new due to `soap` dep updates, without it:

```
> kas-api@0.0.0 build-ts
> tsc
node_modules/soap/lib/wsdl/index.d.ts:1:22 - error TS7016: Could not find a declaration file for module 'sax'. '…/node_modules/sax/lib/sax.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/sax` if it exists or add a new declaration (.d.ts) file containing `declare module 'sax';`

1 import * as sax from 'sax';

Found 1 error in node_modules/soap/lib/wsdl/index.d.ts:1
```

Reply:
Oh, thanks for the reminder. the typescript setup is still a bit broken here, I'll have to fix it.